metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/resolver/FunctionResolver.java

```diff
@@ -43,4 +45,11 @@
    * @param context Context used to initialize.
    */
   void initialize(Context context);
+
+  /**
+   * Perform any cleanup necessary for the loaded Stellar functions.
+   */
```

Comment:
I don't understand why the override here?  Did you mean to default?

Reply:
Default is probably sufficient